x-pack/plugins/ml/server/routes/modules.ts

```diff
@@ -217,9 +466,53 @@ export function dataRecognizer({ router, mlLicense }: RouteInitialization) {
    *
    * @api {post} /api/ml/modules/jobs_exist/:moduleId Check if module jobs exist
    * @apiName CheckExistingModuleJobs
-   * @apiDescription Checks if the jobs in the module have been created.
+   * @apiDescription Check whether the jobs in the module with the specified ID exist in the
+   *      current list of jobs. The check runs a test to see if any of the jobs in existence
+   *      have an ID which ends with the ID of each job in the module. This is done as a prefix
+   *      may be supplied in the setup endpoint which is added to the start of the ID of every job in the module.
    *
-   * @apiParam {String} moduleId Module id
+   * @apiParam {String} moduleId ID of the module to check whether the jobs defined in the module exist.
```

Comment:
`schema.object(getModuleIdParamSchema())` also can be defined as module in schemas

Reply:
Done in https://github.com/elastic/kibana/pull/66738/commits/7a6fd145038f2be82eebe24681815f7ca03d38cc